http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/TestLeoRoutes.scala

```diff
@@ -125,4 +127,14 @@ trait TestLeoRoutes { this: ScalatestRouteTest with Matchers with ScalaFutures w
     cookie.domain shouldBe None
     cookie.path shouldBe Some("/")
   }
+
+  // TODO: remove when we upgrade to akka-http 10.2.0.
+  // See comment in CookieHelper.setTokenCookie.
+  private[api] def validateRawCookie(setCookie: Option[HttpHeader],
+                                     expectedCookie: HttpCookiePair = tokenCookie,
+                                     age: Long = tokenAge): Unit = {
+    setCookie shouldBe 'defined
+    setCookie.get.name shouldBe "Set-Cookie"
+    setCookie.get.value shouldBe s"${expectedCookie.name}=${expectedCookie.value}; Max-Age=${age.toString}; Path=/; Secure; SameSite=None"
```

Comment:
The previous version of this rounded `maxAge` to the nearest 10 before validation, to avoid test clock issues. May need to do something similar here if tests fail intermittently.